eras/alonzo/impl/src/Cardano/Ledger/Alonzo/TxBody.hs

```diff
@@ -26,19 +26,19 @@ module Cardano.Ledger.Alonzo.TxBody
     DataHash32,
     AlonzoTxBody
       ( AlonzoTxBody,
-        inputs,
-        collateral,
-        outputs,
-        txcerts,
-        txwdrls,
-        txfee,
-        txvldt,
-        txUpdates,
-        reqSignerHashes,
-        mint,
-        scriptIntegrityHash,
-        adHash,
-        txnetworkid
+        atbInputs,
+        atbCollateral,
+        atbOutputs,
+        atbCerts,
+        atbWdrls,
+        atbTxFee,
+        atbValidityInterval,
+        atbUpdate,
+        atbReqSignerHashes,
+        atbMint,
+        atbScriptIntegrityHash,
+        atbAuxDataHash,
+        atbTxNetworkId
```

Comment:
Adding these renames to the changelog would probably be a good idea.

Reply:
Oops! Missed it.